packages/storage/src/providers/s3/apis/uploadData/putObjectJob.ts

```diff
@@ -48,6 +49,7 @@ export const putObjectJob =
 			onProgress,
 		} = uploadDataOptions ?? {};
 
+		const ChecksumCRC32 = await calculateContentCRC32(data);
```

Comment:
Nit:

```suggestion
		const checksumCRC32 = await calculateContentCRC32(data);
```

Dumb question: why isn't this one gated like in `uploadPartExecutor`? Ditto about consolidating Md5 calculation logic up here